README.md

```diff
@@ -28,43 +28,75 @@ Please read the [Contribution Guide](CONTRIBUTING.md) before submitting any pull
 ## Introduction
 This documentation describes the backend used for the SwissCovid application. It is focused on providing information for the requests used for the Exposure Notification framework. Although, code of the old format is still provided, no documentation or support is available except the code itself.
 
+## Federation Gateway Service
+### Introduction 
+Since multiple countries are now using the Exposure Notifications Framework, the European Union started an effort to allow federation of keys coming from multiple countries. Every country connected to the gateway can then up- and download keys from all participating countries (relayed via the gateway). This means that people e.g. using the Swiss app in Germany can, if they decide to share the keys with Germany, infect people in Germany and vice versa.
```

Comment:
... if they decide to share
- the
+ their
keys with Germany, infect people
- in Germany
+ using the German app
and vice versa.
-----
it actually has nothing to do with which country people are in, only with which app they're using

Reply:
👍